static/locales/en-US.yaml

```diff
@@ -406,11 +406,15 @@ Settings:
   Player Settings:
     Player Settings: Player
     Play Next Video: Play Next Video
-    Turn on Subtitles by Default: Turn on Subtitles by Default
+    Turn on Subtitles by Default: Enable Subtitles by Default
     Autoplay Videos: Autoplay Videos
     Proxy Videos Through Invidious: Proxy Videos Through Invidious
     Autoplay Playlists: Autoplay Playlists
-    Enable Theatre Mode by Default: Enable Theatre Mode by Default
+    Default Viewing Mode:
+      Default Viewing Mode: Default Viewing Mode
```

Comment:
Should Theater mode and Default be added here? 

Also maybe a weird question to ask but should it be called Theater mode or Theater? I mean the header of the dropdown is called Default Viewing Mode so the user does already know we are talking about Modes here. 

![image](https://github.com/user-attachments/assets/250b5aac-39c5-4cf5-8afd-2893b7f90352)